tests/v1/kv_connector/nixl_integration/run_accuracy_test.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+set -xe
+
+# Models to run
+MODELS=(
+    "Qwen/Qwen3-0.6B"
+    "deepseek-ai/deepseek-vl2-tiny"
```

Comment:
This turns out to not actually have MLA very unfortunately -- maybe we go with `mgoin/DeepSeek-Coder-V2-Lite-Instruct-FP8`?

Reply:
in fup